src/mgmt/osp-sah.ks

```diff
@@ -366,18 +363,18 @@ do
   echo "nameserver ${ns}" >> /etc/resolv.conf
 done
 
-# Configure the ntp daemon
-systemctl enable ntpd
-sed -i -e "/^server /d" /etc/ntp.conf
+# Configure the chrony daemon for ntp
+systemctl enable chronyd
+sed -i -e "/^server /d" /etc/chrony.conf
 
 for ntps in ${NTPServers//,/ }
 do
-  echo "server ${ntps}" >> /etc/ntp.conf
+  echo "server ${ntps}" >> /etc/chrony.conf
```

Comment:
Check to ensure pool isn't the default vs server

Reply:
Verified, pool and server are both valid. Pool in case of a range of servers and server in case of individual hosts